internal/pkg/service/stream/definition/by.go

```diff
@@ -0,0 +1,42 @@
+package definition
+
+import (
+	"strconv"
+
+	"github.com/keboola/go-client/pkg/keboola"
+)
+
+const (
+	BySystem = "system"
+	ByUser   = "user"
+)
+
+type ByType string
+
+// By describes actor of an operation.
+type By struct {
+	Type      ByType `json:"type" validate:"required,oneof=system user"`
+	TokenID   string `json:"tokenId,omitempty"`
+	TokenDesc string `json:"tokenDesc,omitempty"`
+	UserID    string `json:"userId,omitempty"`
+	UserName  string `json:"userName,omitempty"`
+}
+
```

Comment:
Added new `By` type to track actor of the change: `CreatedBy`, `CreatedAt`, ... `DeletedAt`, ...